c7n/resources/account.py

```diff
@@ -424,6 +425,86 @@ def process(self, resources, event=None):
         return []
 
 
+@actions.register('set-password-policy')
+class SetAccountPasswordPolicy(BaseAction):
+    """Set an account's password policy.
+
+    This only changes the policy for the items provided.
+    If this is the first time setting a password policy and an item is not provided it will be
+    set to the defaults defined in the boto docs for IAM.Client.update_account_password_policy
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: set-account-password-policy
+                resource: account
+                filters:
+                  - not:
+                    - type: password-policy
+                      key: MinimumPasswordLength
+                      value: 10
+                      op: ge
+                actions:
+                    - type: set-password-policy
+                      policy:
+                        - key: MinimumPasswordLength
```

Comment:
why not just make this a mapping?

Reply:
A mapping could work, I wanted to set json schema to validate the key against a list of enums but gave up after playing around for awhile. I'll revisit this afternoon to see if I can figure it out. This way there is not k:v but something like MinimumPasswordLength: 12, right?

Reply:
with this as an array your converting to a dict to validate and make an api call, just use a mapping and skip the conversion.

Reply:
Ah yes I see why thats also kind of superfluous now. Thanks! Updated in ba13962.